src/ui/Modal/Modal.spec.tsx

```diff
@@ -62,7 +61,7 @@ describe('Modal', () => {
         />
       )
 
-      await user.click(screen.getByLabelText('Close'))
+      await userEvent.click(screen.getByLabelText('Close'))
```

Comment:
Can you bring the `userEvent.setup()` we ran into issues with using it directly previously, it's also the preferred style outlined by [testing-library](https://testing-library.com/docs/user-event/setup).

Reply:
I see, thanks for the context.